app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PostsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index; end
+
+  def show
+    @post = Post.find params[:id]
+  end
+
+  def create
+    @post = Post.new(message: params[:msg])
```

Comment:
This should be `params[:message]` not `params[:msg]`. Additionally, you should also take a look at using strong parameters to prevent mass assignment from end users. https://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html